packages/@react-native-windows/automation/src/AutomationEnvironment.ts

```diff
@@ -50,7 +51,16 @@ class WinAppDriverEnvironment extends NodeEnvironment {
 
     this.winappdriverBin =
       passedOptions.winAppDriverBin ||
-      'C:\\Program Files (x86)\\Windows Application Driver\\WinAppDriver.exe';
+      path.join(
+        process.env['PROGRAMFILES(X86)']!,
+        'Windows Application Driver\\WinAppDriver.exe',
+      );
```

Comment:
This was from a different PR I've had in draft, but worth fixing if this is going out further.